src/set_theory/cardinal/finite.lean

```diff
@@ -41,6 +41,23 @@ cardinal.to_nat_congr f
 lemma card_eq_of_bijective (f : α → β) (hf : function.bijective f) : nat.card α = nat.card β :=
 card_congr (equiv.of_bijective f hf)
 
+lemma card_le_of_injective [finite β] (f : α → β) (h : function.injective f) :
+  nat.card α ≤ nat.card β :=
+begin
+  haveI := fintype.of_finite β,
+  haveI := fintype.of_injective f h,
+  simp_rw [nat.card_eq_fintype_card, fintype.card_le_of_injective f h],
+end
+
+lemma card_le_of_surjective [finite α] (f : α → β) (h : function.surjective f) :
+  nat.card β ≤ nat.card α :=
```

Comment:
This is https://leanprover-community.github.io/mathlib_docs/data/finite/card.html#finite.card_le_of_surjective